src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -22,7 +23,11 @@ public override IEnumerable<IOperation> Children
         {
             get
             {
-                yield return Reference;
+                IOperation current;
+                if ((current = Reference) != null)
```

Comment:
>(current = Reference) [](start = 20, length = 21)

Is there a benefit in capturing Reference in a local? If so, please move the assignment to the declaration.

Reply:
In general, we are trying to avoid using inline assignments, unless they lead to a substantial simplification.

---
In reply to: [145212748](https://github.com/dotnet/roslyn/pull/22726#discussion_r145212748) [](ancestors = 145212748)

Reply:
working on this.